admin/tabs/userData/JsonBlobModel.js

```diff
@@ -135,7 +136,9 @@ export class JsonBlobModel extends HoistModel {
             {field: 'value'},
             {field: 'meta'},
             {field: 'archived'},
-            {field: 'archivedDate'},
+            {field: 'archivedDate', formField: {readonlyRenderer: v => {
+                return (!isDate(v) || v.getTime() === 0) ? '-' : fmtDateTime(v);
+            }}},
```

Comment:
`archivedDate` appears to be a bit of a funny field on `JSONBlob`, presumably for good reasons. This is intended to be a relatively safe / robust way to avoid displaying the epoch start date when `0` - without needing to touch anything else. 😃